src/notification-preferences/data/constants.js

```diff
@@ -0,0 +1,3 @@
+const NOTIFICATION_CHANNELS = ['web'];
+
+export default NOTIFICATION_CHANNELS;
```

Comment:
since this file will hold multiple constants, i think we should not export this constant as default
```suggestion
export const NOTIFICATION_CHANNELS = ['web'];
```

Reply:
We have just this constant right now and maybe in the future, it will remain the same. Apart from that eslint prefers to single export as default. If I make it a name export then I need to disable eslint which is not recommended.